reflex/components/radix/themes/layout/list.py

```diff
@@ -87,32 +90,37 @@ def add_style(self) -> dict[str, Any] | None:
             "direction": "column",
         }
 
+    def _exclude_props(self) -> list[str]:
+        return ["items", "list_style_type"]
+
 
 class UnorderedList(BaseList, Ul):
     """Display an unordered list."""
 
     tag = "ul"
 
+    # The style of the list.
+    list_style_type: LiteralListStyleTypeUnordered = "disc"
```

Comment:
this can be removed and inherited from the parent; the default is already being grabbed in the `create` function

(same with below for `OrderedList`)